vendor/vendor.json

```diff
@@ -441,28 +441,28 @@
 			"revisionTime": "2017-04-20T09:04:30Z"
 		},
 		{
-			"checksumSHA1": "vrp0KSG3kZimXujiJMc/NTf2w3M=",
+			"checksumSHA1": "+Yj9sWqspkwTTYsSpnDACT7ZPU0=",
```

Comment:
That's a good idea, but this is how it was for now. What is more in master this might be tricky unless we will be able to have master here and always take latest artifact library changes.